lib/test.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Need an actual working test validating core functionality in order to merge a PR